lib/timers.js

```diff
@@ -82,11 +82,11 @@ function listOnTimeout() {
 
   debug('timeout callback ' + msecs);
 
-  var now = Timer.now();
-  debug('now: ' + now);
-
   var first;
   while (first = L.peek(list)) {
+    var now = Timer.now();
```

Comment:
Maybe we should do it only if the list was changed? I believe this will come with a performance loss for a many-timer case.